tribits/ci_support/CDashQueryAnalyzeReport.py

```diff
@@ -578,16 +578,18 @@ def getTestsWtihIssueTrackersListFromCsvFile(testsWithIssueTrackersFile):
 # Write list of tests from a Tests LOD to a CSV file structure meant to match
 # tests with issue trackers CSV file.
 #
-def writeTestsLODToCsvFileStructure(testsLOD):
+def writeTestsLODToCsvFileStructure(testsLOD,
+    issueTrackerUrl="", issueTracker="",
+  ):
   csvFileHeadersList = copy.deepcopy(g_testsWithIssueTrackersCsvFileHeadersRequired)
   csvFileRowsList = []
   for testDict in testsLOD:
     csvFileRow = (
       testDict['site'], 
       testDict['buildName'], 
       testDict['testname'],
-      "",  # issue_tracker_url
-      "",  # issue_tracker
+      issueTrackerUrl,  # issue_tracker_url
+      issueTracker,  # issue_tracker
```

Comment:
You still want the comments here?

Reply:
Well, those are the names of the actual column headers.  That is why those comments exist.